xun/functions/transformations.py

```diff
@@ -540,7 +545,7 @@ def visit_Call(self, node):
             store_accessor_load_call = ast.Call(
                 func=store_accessor_load_func,
                 args=[call_node],
-                keywords=[],
+                keywords=[ast.keyword('hash', hash_expr)],
```

Comment:
[github is stupid, so new comment]
Can you test that correct hash is used or is it too specific case?
If I remove the keyword thing now, nothing fails.